Doc/library/faulthandler.rst

```diff
@@ -52,6 +52,8 @@ Dumping the traceback
    Dump the tracebacks of all threads into *file*. If *all_threads* is
    ``False``, dump only the current thread.
 
+   See also :func:`traceback.print_tb` which can be used to print a traceback object.
```

Comment:
```suggestion
   .. seealso:: :func:`traceback.print_tb`, which can be used to print a traceback object.
```

Using the standard directive is appropriate and conventional here, too. Also, add a missing comma.